lib/cli/src/commands/create_exe.rs

```diff
@@ -0,0 +1,273 @@
+//! Create a standalone native executable for a given Wasm file.
+
+use crate::store::{CompilerOptions, EngineType};
+use anyhow::{Context, Result};
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use structopt::StructOpt;
+use wasmer::*;
+
+const WASMER_MAIN_C_SOURCE: &[u8] = include_bytes!("wasmer_create_exe_main.c");
+
+// TODO: to get this working locally I had to add wasmer_wasm.h and wasm.h to `.wasmer/include` manually.
+// this needs to be fixed before we can ship this.
```

Comment:
Make sure we ship `wasmer_wasm.h` and `wasm.h` before merging

Reply:
It's on the todo list at the top of the issue (you may need to refresh the page to get Github to show you that though)